packages/dashboard/pages/api/did/[did].ts

```diff
@@ -0,0 +1,12 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
+import getElement from '../../../server/getElement';
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<any>
+) {
+  const { did } = req.query;
+  const element = await getElement();
```

Comment:
I think there is a decorator for this... 

https://github.com/transmute-industries/sidetree.js/blob/main/packages/dashboard/pages/api/1.0/identifiers/%5Bdid%5D/index.tsx#L27

https://github.com/transmute-industries/sidetree.js/blob/main/packages/dashboard/middleware/sidetree.ts